CMakeLists_files.cmake

```diff
@@ -75,4 +75,5 @@ list (APPEND TEST_DATA_FILES
         tests/testRFT.DATA
         tests/table_deck.DATA
         tests/summary_deck_non_constant_porosity.DATA
+        tests/summary_eff_fac.DATA
```

Comment:
Please rename it `summary_eff_fac.data` or `SUMMARY_EFF_FAC.DATA`. We have many of these mixed case test decks, and that is borderline legal input.

Reply:
We'll fix it!